frontend/components/ImagesSection.js

```diff
@@ -0,0 +1,35 @@
+import { View, Image } from 'react-native'
+import React, { useEffect, useState } from 'react'
+import ImagesCarousel from './ImagesCarousel'
+import useAPI from '../hooks/useAPI'
+
+export default function ImagesSection({ note_id }) {
+    const [images, setImages] = useState([]);
+    const { loading, fetchData } = useAPI(`notes/getimages/${note_id}`, ({ images }) => setImages(images), console.log);
```

Comment:
we use this hook here
we pass the url parameter and functions to run: onSuccess (we set images if we got them successfully) and onError(we print this error if we got one)
we get loading (true = there is no result of the request yet, false = we got either result or error)
fetchData is used to actually trigger request